std/range/package.d

```diff
@@ -699,8 +700,7 @@ pure @safe nothrow unittest
 pure @safe nothrow @nogc unittest
 {
     int[4] testArr = [1,2,3,4];
-    //just checking it compiles
-    auto s = testArr[].stride(2);
+    static assert(__traits(compiles,testArr[].stride(2)));
```

Comment:
doing so may cause a GC allocation which makes the unittest drop the @nogc. Since the test is not public, maybe it would be better to leave it like this